module/mock/hot_stuff_follower.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by mockery v1.0.0. DO NOT EDIT.
```

Comment:
why are we regenerating this?

Reply:
I think I just wanted to make sure that all Makefile options still work cloning the observer tests from access. I can remove it from the change.

Reply:
Fixed in: https://github.com/GetElastech/flow-go/commit/5078c8c76b61ec4fb9acf3caefecb365c358b1ba